hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -142,9 +159,13 @@ public void triggerContainerHealthCheck() {
               " processing {} containers.", Time.monotonicNow() - start,
           containers.size());
       logUnhealthyContainerStats(unhealthyContainerStateStatsMap);
-      processedContainers.clear();
-    } finally {
-      lock.writeLock().unlock();
+      if (containers.size() > FETCH_COUNT) {
+        startID = ContainerID.valueOf(
```

Comment:
startID should be "lastID + 1", right?

Reply:
Yes, will correct that, startId is inclusive.